src/main/java/com/dataartisans/flink/dataflow/translation/wrappers/streaming/FlinkGroupAlsoByWindowWrapper.java

```diff
@@ -491,6 +516,9 @@ public BoundedWindow window() {
 
 				@Override
 				public void outputWindowedValue(KV<K, VOUT> output, Instant timestamp, Collection<? extends BoundedWindow> windows, PaneInfo pane) {
+					// TODO: No need to represent timestamp twice.
+					// collector.setAbsoluteTimestamp(timestamp.getMillis());
+					collector.setAbsoluteTimestamp(0);
 					collector.collect(WindowedValue.of(output, timestamp, windows, pane));
```

Comment:
I think we have to correctly set the timestamp here. It needs to be set in the collector and the WindowedValue. Setting it to 0 would probably cause wrong results.